src/main/java/picard/sam/util/ReadOutputCommandLineProgram.java

```diff
@@ -0,0 +1,14 @@
+package picard.sam.util;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import picard.cmdline.CommandLineProgram;
+
+/**
+ * Abstract class that holds parameters and methods common to classes that want to add PG tags to reads in SAM/BAM files
+ *
+ */
+public abstract class ReadOutputCommandLineProgram extends CommandLineProgram {
```

Comment:
I wasn't sure what would be a good name either.  This is something I discussed with @jacarey as well